src/browser_action/browser_action.html

```diff
@@ -8,7 +8,7 @@
 	<title>ETHSBell</title>
 	<link rel="stylesheet" href="ethsbell.css" />
 	<meta http-equiv="Content-Security-Policy"
-		content="default-src 'self'; connect-src 'self' https://ethsbell.app/;" />
+		content="default-src 'self'; connect-src 'self' https://ethsbell.app/ https://dayschool.ethsbell.app/;" />
```

Comment:
Does it make more sense to just do *.ethsbell.app?

Reply:
> Does it make more sense to just do *.ethsbell.app?

 Nope. Rather than allowing any requests to any subdomain and the root domain, this only allows requests to any subdomain. 
This breaks the page for people using the main instance.

source:
![image](https://user-images.githubusercontent.com/37093293/221423851-3dc1db28-108d-4a47-9a15-d31e3f4ff37c.png)